packages/dd-trace/test/plugins/helpers.js

```diff
@@ -10,6 +10,7 @@ function expectSomeSpan (agent, expected, timeout) {
     const scoredErrors = []
     for (const trace of traces) {
       for (const span of trace) {
+        // console.log(expected, span)
```

Comment:
What's our stance on commented out code?

Reply:
Stance is they should be removed, which I will do right now.